health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/SetupMicroplan.js

```
@@ -372,10 +377,10 @@ const SetupMicroplan = ({ hierarchyType, hierarchyData }) => {
         label={getNextActionLabel()}
       />
       {setupCompleted ? (
-        <ActionBar style={{ zIndex: "19" }} setactionFieldsToRight
-          actionFields={[
-            <Button label={t("GO_BACK_TO_MY_MICROPLAN")} onClick={() => history.goBack()} />
-          ]}
+        <ActionBar
+          style={{ zIndex: "19" }}
+          setactionFieldsToRight
+          actionFields={[<Button label={t("GO_BACK_TO_MY_MICROPLAN")} onClick={() => history.goBack()} />]}
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)